.github/workflows/test_workflow_scripts/golang-mysql-linux.sh

```diff
@@ -12,6 +12,13 @@ fi
 
 rm -rf keploy/
 
+# Generate the keploy-config file.
+sudo ./../../keployv2 config --generate
+
+# Update the global noise to updated_at.
+config_file="./keploy.yml"
+sed -i 's/global: {}/global: {"body": {"updated_at":[]},"header" : {"Content-Length" : []}}/' "$config_file"
```

Comment:
Content Length can never come in the noisy field, as noisy fields are those which are to be ignored while replaying, if the content length itself doesnt match for example- the length of the response ,we cant process it further. L